.github/workflows/javascript.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: javascript
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  test:
+    name: Node ${{ matrix.node-version }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16.x]
+        os: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: npm install
+      run: npm i
+    - name: npm run lint
+      run: npm run lint
```

Comment:
As suggested in https://github.com/os-autoinst/openQA/pull/4042/files#r668812082 please provide something that can be called by users and CI agnostic. We don't want to be dependant on any specific CI service too much, regardless if it's travisCI, circleCI or GHA so better keep github actions as slim as possible.

Reply:
The integration would be nice to have but yes, a generic script should be provided in any case.

Reply:
I'll add a script for local development, but the GitHub Action can't be changed much, since `actions/setup-node` (which is made by GitHub) provides the GitHub annotation features.

Reply:
The script will probably be as simple as:
```sh
if ! command -v npm &> /dev/null
then
    echo "Please install node to use the js-tidy script"
    exit
fi
npm install
npm run fix
```